en/docs/setup/migrating-preparing-for-migration.md

```diff
@@ -94,13 +94,17 @@ All the other data will not be preserved in the new system.
 
 Now let's see how to do the blue-green deployment with WSO2 Identity Server.
 
+!!! info
+    Note that this **data sync tool** is only recommended for use with MySQL and Oracle databases
+    since it has been tested with MySQL and Oracle only.
```

Comment:
```suggestion
    Note that this **data sync tool** is only recommended for use with MySQL and Oracle databases
    since it has only been tested with MySQL and Oracle.
```

Reply:
Addressed with https://github.com/wso2/docs-is/pull/2484/commits/045ea4b59c6d26a54259cca970f1c2f55a866558